pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/FunctionsBase.java

```diff
@@ -94,14 +183,106 @@ public void registerFunction(final @PathParam("tenant") String tenant,
+                            "deadLetterTopic  \n" +
+                            "  The topic where all messages which could not be processed successfully are sent  \n" +
+                            "runtimeFlags  \n" +
+                            "  Any flags that you want to pass to the runtime. note that in thread mode, these flags will have no impact  \n" +
+                            "fqfn  \n" +
+                            "  The Fully Qualified Function Name (FQFN) for the function  \n" +
+                            "inputSpecs  \n" +
+                            "  A generalized way of specifying inputs  \n" +
+                            "inputs  \n" +
+                            "  The function's input topic or topics (multiple topics can be specified as a comma-separated list)  \n" +
+                            "jar  \n" +
+                            "  Path to the jar file for the function (if the function is written in Java). " +
+                            "  It also supports url-path [http/https/file (file protocol assumes that file " +
+                            "  already exists on worker host)] from which worker can download the package.  \n" +
+                            "py  \n" +
+                            "  Path to the main Python file/Python Wheel file for the function (if the function is written in Python)  \n" +
+                            "go  \n" +
+                            "  Path to the main Go executable binary for the function (if the function is written in Go)  \n" +
+                            "logTopic  \n" +
+                            "  The topic to which the function's logs are produced  \n" +
+                            "maxMessageRetries  \n" +
+                            "  How many times should we try to process a message before giving up  \n" +
+                            "output  \n" +
+                            "  The function's output topic (If none is specified, no output is written)  \n" +
+                            "outputSerdeClassName  \n" +
+                            "  The SerDe class to be used for messages output by the function  \n" +
+                            "parallelism  \n" +
+                            "  The function's parallelism factor (i.e. the number of function instances to run)  \n" +
+                            "processingGuarantees  \n" +
+                            "  The processing guarantees (aka delivery semantics) applied to the function" +
+                            "  Possible Values: [ATLEAST_ONCE, ATMOST_ONCE, EFFECTIVELY_ONCE]  \n" +
+                            "retainOrdering  \n" +
+                            "  Function consumes and processes messages in order  \n" +
+                            "outputSchemaType  \n" +
+                            "   Represents either a builtin schema type (eg: 'avro', 'json', ect) or the class name for a Schema implementation" +
```

Comment:
```suggestion
                            "   Represents either a builtin schema type (for example, 'avro', 'json', ect) or the class name for a Schema implementation" +
```

Don't use "e.g." in a formal text.